src/Maestro/Maestro.Web/Startup.cs

```diff
@@ -277,21 +292,52 @@ private void ConfigureApi(IApplicationBuilder app)
         {
             app.UseExceptionHandler(ConfigureApiExceptions);
 
-            app.UseAuthentication();
+            app.UseSwagger(
```

Comment:
>app.UseSwagger( [](start = 12, length = 15)

This `UseSwagger` is too high. If you want to move this here the handler directly above it that forwards `/api/swagger.json` to `/api/{latestVersion}/swagger.sjon` needs to come before this one.

Reply:
Moved the other thing up (it needs to come before the Routing/Endpoints stuff.